sympy/stats/crv.py

```diff
@@ -335,13 +335,25 @@ def where(self, condition):
         interval = interval.intersect(self.domain.set)
         return SingleContinuousDomain(rv.symbol, interval)
 
+
     def conditional_space(self, condition, normalize=True, **kwargs):
 
         condition = condition.xreplace(dict((rv, rv.symbol) for rv in self.values))
-
         domain = ConditionalContinuousDomain(self.domain, condition)
+
         if normalize:
-            pdf = self.pdf / domain.integrate(self.pdf, **kwargs)
+            # create a clone of the variable to
+            # make sure that variables in nested integrals are different
+            # from the variables outside the integral
+            # this makes sure that they are evaluated separately
+            # and in the correct order
+            to_dummy = lambda rv: Dummy(str(rv.symbol))
+            replacement = {rv.symbol: to_dummy(rv) for rv in self.values}
```

Comment:
Could we use `self.symbols` instead of `self.values`?

Reply:
I think so. Thanks for spotting what I missed! Fixing this and running another test. Will add another commit once it's done running.